canonical/ubuntu.com
# feat: add Qualcomm IoT page

## Done

- Add Qualcomm IoT page and contact form.

Copy doc for the webpage: https://docs.google.com/document/d/1BP2T63R6kdnUFQFIZcbXbTIgY-9fQVgVRLsSUU5Rvx8/edit?tab=t.0
Copy doc for the form: https://www.google.com/url?q=https://docs.google.com/document/d/1vZepBdXUWbidF3BD1zwvudiykRjtItHT4ssCqj6Dh-U/edit?tab%3Dt.0&sa=D&source=docs&ust=1731909684091428&usg=AOvVaw3Gjv_xqSPKCnQoYfDiXgwx

## QA

- Open the demo https://ubuntu-com-14500.demos.haus/
- Check that there is a link to the new page:
 -- In the top nav click 'Get Ubuntu'.
 -- To the side click 'Ubuntu for IoT'
 -- Under 'Quick links' there should be a link: 'Install Ubuntu on Qualcomm IoT Platforms'.
- Visit https://ubuntu-com-14500.demos.haus/download/qualcomm-iot
- Click 'Get in touch' and the new form should appear.

## Issue / Card

https://warthogs.atlassian.net/browse/WD-16036

## Screenshots

![iot](https://github.com/user-attachments/assets/73e484c9-1464-403b-8282-12740758d0df)

![Screenshot 2024-11-26 at 14-48-26 Install Ubuntu on Qualcomm IoT Platforms Ubuntu](https://github.com/user-attachments/assets/ced7910a-b696-4ac9-b54e-78618c413f9e)
